hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -120,38 +135,48 @@
                 <relocation>
                   <pattern>org</pattern>
                   <shadedPattern>${shaded.prefix}.org</shadedPattern>
-                  <includes>
-                    <include>org.yaml.**.*</include>
-                    <include>org.sqlite.**.*</include>
-                    <include>org.tukaani.**.*</include>
-                    <include>org.bouncycastle.**.*</include>
-                    <include>org.rocksdb.**.*</include>
-                    <include>org.apache.commons.cli.**.*</include>
-                    <include>org.apache.commons.compress.**.*</include>
-                    <include>org.apache.commons.codec.**.*</include>
-                    <include>org.apache.commons.beanutils.**.*</include>
-                    <include>org.apache.commons.collections.**.*</include>
-                    <include>org.apache.commons.digester.**.*</include>
-                    <include>org.apache.commons.io.**.*</include>
-                    <include>org.apache.commons.logging.**.*</include>
-                    <include>org.apache.commons.validator.**.*</include>
-                    <include>org.apache.commons.lang3.**.*</include>
-                    <include>org.sqlite.**.*</include>
-                    <include>org.apache.thrift.**.*</include>
-                  </includes>
+                  <excludes>
+                    <exclude>org.apache.commons.logging.**</exclude>
+                    <exclude>org.apache.hadoop.**</exclude>
+                    <exclude>org.apache.log4j.**</exclude>
+                    <exclude>org.apache.ozone.**</exclude>
+                    <exclude>org.apache.ratis.**</exclude>
+                    <exclude>org.bouncycastle.**</exclude>
+                    <exclude>org.hadoop.ozone.**</exclude>
```

Comment:
`org.hadoop.ozone` is no longer needed after HDDS-12198 (but your branch is 149 commits behind `apache/ozone:master`).

Reply:
Removed the `org.hadoop.ozone`.

Reply:
PS: we have some package name `org.apache.ratis.metrics.xxx`, should we rename it to `org.apache.ozone.ratis.metrics.xxx`

Reply:
> we have some package name `org.apache.ratis.metrics.xxx`

Is it in Ozone?  I don't see it.

Reply:
such as:
https://github.com/apache/ozone/blob/dd74eee7fccd941c7baced88ce75967ef9f269c9/hadoop-hdds/framework/src/main/java/org/apache/ratis/metrics/dropwizard3/RatisMetricsUtils.java#L18

Reply:
Thanks, I missed it.  This one is needed for accessing package-private members.

Reply:
OK, got it.